chain/beacon/node.go

```diff
@@ -341,7 +342,9 @@ func (h *Handler) run(startTime int64) {
 				// channel will trigger again etc until we arrive at the correct
 				// round.
 				go func(c roundInfo, latest *chain.Beacon) {
+					h.l.Debug("beacon_loop", "catchupmode", "last_is", latest.Round, "seep_for", h.conf.Group.CatchupPeriod)
 					h.conf.Clock.Sleep(h.conf.Group.CatchupPeriod)
+					h.l.Debug("beacon_loop", "catchupmode", "last_is", latest.Round, "broadcast")
```

Comment:
We need an even number of key-values in the logs, this has one extra, or it lacks one.

Reply:
```suggestion
					h.l.Debugw("broadcast next partial", "beacon_loop", "catchupmode", "last_is", latest.Round)
```